orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/WebhookResponseProcessor.java

```diff
@@ -87,15 +87,17 @@ public TaskResult process(
   private TaskResult processReceivedHttpStatusException(HttpStatusCodeException e) {
     var webhookOutput = new WebhookStage.WebhookResponseStageData();
 
-    webhookOutput.setStatusCode(e.getStatusCode());
+    webhookOutput.setStatusCode(HttpStatus.valueOf(e.getStatusCode().value()));
```

Comment:
```suggestion
    webhookOutput.setStatusCode(e.getStatusCode());
```
works for me locally.